tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -566,7 +566,8 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
-		[ "$ARCH" == "ppc64le" ] && ([ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu")
+                # LIBC type for ppc64le should be gnu
+		[ "$ARCH" == "ppc64le" ] && LIBC=gnu
```

Comment:
Currently we override LIBC during building the agent. I'll let @fidencio and others comment on this approach of setting it permanently here.

Reply:
Oh, you're right, we do it on the agent side only.
I'm fine about doing it here, I'm totally fine about that, as long as we give a warning of why we're doing this as users may expect to be using the default LIBC option.

Reply:
Thanks! I will update it with a warning.